pkg/kubernetes/client.go

```diff
@@ -240,3 +236,14 @@ func sortWorkerPods(pods *coreV1.PodList) {
 		return strings.Compare(pods.Items[i].ObjectMeta.Name, pods.Items[j].ObjectMeta.Name) < 0
 	})
 }
+
+// BuildKubeClientConfig is used in cmd package
+func BuildKubeClientConfig(masterURL string, kubeConfigPath string, timeout time.Duration) (*restClient.Config, error) {
```

Comment:
Maybe rename this to `BuildClientConfig` since `Kube` is already part of the package this func exists in.

Reply:
Done via 97b1771